alphacsc/utils/dictionary.py

```diff
@@ -19,6 +19,30 @@ def get_D(uv_hat, n_channels):
     return uv_hat[:, :n_channels, None] * uv_hat[:, None, n_channels:]
 
 
+def flip_uv(uv, n_channels):
+    """Ensure the temporal pattern v peak is positive for each atom.
+
+      If necessary, multiply both u and v by -1.
+
+    Parameter
+    ---------
+    uv: array (n_atoms, n_channels + n_times_atom)
+        Rank1 dictionary which should be modified.
+    n_channels: int
+        number of channels in the original multivariate series
+
+    Return
+    ------
+    uv: array (n_atoms, n_channels + n_times_atom)
+    """
+    v = uv[:, n_channels:]
+    index_array = np.argmax(np.absolute(v), axis=1)
+    val_index = np.take_along_axis(v, np.expand_dims(
+        index_array, axis=-1), axis=-1).squeeze(axis=-1)
```

Comment:
```suggestion
    val_index = v[:, index_array]
```

Reply:
maybe use `peak_value` instead of `val_index` which is not super explicit.